src/js/components/agency/visualizations/StatusOfFundsChart.jsx

```diff
@@ -1005,10 +1015,10 @@ const StatusOfFundsChart = ({
                     width={288}
                     styles={!toggle ? {
                         position: 'absolute',
-                        transform: `translate(${mouseValue.x - 144}px,${mouseValue.y - 230}px)`
+                        transform: `translate(${mouseValue.x - 144}px,${mouseValue.y - 270}px)`
```

Comment:
isHovered &&
                <TooltipWrapper
                    className="sof_chart-tt"
                    width={288}
                    styles={!toggle ? {
                        position: 'absolute',
                        transform: `translate(${mouseValue.x - 144}px,${mouseValue.y - 230}px)`
the above is a snippet obviously, but produces the below image for me, does it not look like that for you??
![image](https://user-images.githubusercontent.com/102241022/227006403-86530828-561b-42d5-b7cb-e49e8ecc3af5.png)

Reply:
It does until I click into the next level
Then it changes and does the block I was mentioning before hand

Reply:
got it, looked at it more and what you'll have to do is based on the level (which is a prop you have access to) and based on the level you'll have to return a slightly different number if you're at a level greater than 0
obviously you know ${mouseValue.y - 230}px works fine for the default case 

you'll also have to check how it looks when the toggle is also enabled, let me know if that doesn't make sense